modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -54,7 +52,7 @@ def _read(cls, filepath_or_buffer, **kwargs):
             Query compiler with imported data for further processing.
         """
         # Ensures that the file is a string file path. Otherwise, default to pandas.
-        filepath_or_buffer = cls.get_path_or_buffer(filepath_or_buffer)
+        filepath_or_buffer = cls.get_path_or_buffer(stringify_path(filepath_or_buffer))
```

Comment:
`stringify_path` is needed here because `get_path` isn't called?

Reply:
`get_path` in the glob file reader returns a list of files, so `stringify_path` makes sense here.